src/Kdyby/Doctrine/DI/OrmExtension.php

```diff
@@ -720,8 +727,9 @@ protected function processRepositories()
 		}
 
 		foreach ($this->configuredManagers as $managerName => $_) {
-			$builder->getDefinition($this->prefix($managerName . '.repositoryFactory'))
-				->addSetup('setServiceIdsMap', [
+			/** @var Nette\DI\ServiceDefinition $def */
+			$def = $builder->getDefinition($this->prefix($managerName . '.repositoryFactory'));
+			$def->addSetup('setServiceIdsMap', [
```

Comment:
why?

Reply:
phpstan, same problem as above.
If you know a better fix, please suggest.
We can always whitelist this error message in phpstan, of course.